CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
-## [_Unreleased_](https://github.com/freckle/github-app-token/compare/v0.0.1.1...main)
+## [_Unreleased_](https://github.com/freckle/github-app-token/compare/v0.0.1.2...main)
 
-## [v0.0.1.0](https://github.com/freckle/github-app-token/tree/v0.0.1.1)
+## [v0.0.1.0](https://github.com/freckle/github-app-token/compare/v0.0.1.0...v0.0.1.1)
+
+- Implement refresh wake-ups more efficiently
+
+## [v0.0.1.0](https://github.com/freckle/github-app-token/compare/v0.0.0.1...v0.0.1.1)
```

Comment:
Or this,

```suggestion
## [v0.0.1.1](https://github.com/freckle/github-app-token/compare/v0.0.0.1...v0.0.1.1)
```

I really hate these manual CHANGELOGs. I've tried so many "better" ways, but just can't make them stick. Some day.